ajax/libs/event-drops/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "event-drops",
+  "filename": "index.min.js",
+  "description": "A time based / event series interactive visualization using d3.js. Use drag and zoom to navigate in time.",
+  "author": "Thiery Michel <thiery@marmelab.com>",
+  "keywords": [
+    "dom",
+    "w3c",
+    "visualization",
+    "svg",
+    "animation",
+    "canvas"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marmelab/EventDrops"
+  },
+  "license": "MIT",
+  "autoupdate": {
```

Comment:
Hi @austin665, I think the files we need are in `dist` in npm package. Could you remove the `git auto-update` and change to use:
```
"npmFileMap": [
  {
    "basePath": "dist",
    "files": [
      "**/*"
    ]
  }
]
```